### PR TITLE
make sync_slurm_acc work for brussel

### PR DESCRIPTION
**Pull request by smoors**
depends on https://github.com/hpcugent/vsc-config/pull/128

### CODE REVIEW

File: bin/sync_slurm_acct.py [hunk 1]
```diff
@@ -71,9 +71,10 @@ def main():
             "store",
             "https://apivsc.ugent.be/django",
         ),
+        'host_institute': ('Name of the institute where this script is being run', str, 'store', GENT),
         "clusters": (
             "Cluster(s) (comma-separated) to sync for. "
-            "Overrides GENT_SLURM_COMPUTE_CLUSTERS that are in production.",
+            "Overrides <host_institute>_SLURM_COMPUTE_CLUSTERS that are in production.",
```
Comment (stdweird) [inline bin/sync_slurm_acct.py:77]: some missing templating?

Comment (smoors) [reply bin/sync_slurm_acct.py:77]: uh no, this is intentional..

File: setup.py [hunk 2]
```diff
@@ -24,7 +24,7 @@
 from vsc.install.shared_setup import ag, jt
 
 PACKAGE = {
-    'version': '2.3.1',
+    'version': '2.3.2',
```
Comment (stdweird) [inline setup.py:27]: bump the vsc-config minimum version here as well

File: bin/sync_slurm_acct.py (hunk 1 above)
Comment (stdweird) [reply bin/sync_slurm_acct.py:77]: ok, i see.

File: setup.py (hunk 2 above)
Comment (smoors) [reply setup.py:27]: ok, done.

File: setup.py [hunk 3]
```diff
@@ -35,7 +35,7 @@
     'install_requires': [
         'vsc-accountpage-clients >= 2.0.0',
         'vsc-base >= 3.0.6',
-        'vsc-config >= 3.3.3',
+        'vsc-config >= 3.3.6',
```
Comment (wpoely86) [inline setup.py:38]: can you change this to 3.4.2?

Comment (smoors) [reply setup.py]: ok, done

**Comment by smoors** [comment]
@stdweird any idea why the test fails?

File: lib/vsc/administration/slurm/sync.py [hunk 4]
```diff
@@ -267,7 +267,7 @@ def slurm_institute_accounts(slurm_account_info, clusters):
     return commands
 
 
-def slurm_vo_accounts(account_page_vos, slurm_account_info, clusters):
+def slurm_vo_accounts(account_page_vos, slurm_account_info, clusters, host_institute=GENT):
```
Comment (stdweird) [inline lib/vsc/administration/slurm/sync.py:270]: make `host_institute` positional arg

**Comment by stdweird** [comment]
@itkovian looks ok for me

**Comment by stdweird** [comment]
@smoors @wpoely86 tests fail with same error as #103

File: lib/vsc/administration/slurm/sync.py (hunk 4 above)
Comment (smoors) [reply lib/vsc/administration/slurm/sync.py:270]: @stdweird so you do not want a default value for `host_institute`?
should I do the same for `slurm_institute_accounts()`?

Comment (stdweird) [reply lib/vsc/administration/slurm/sync.py:270]: the default in the options is enough
i don't understand the 2nd question

Comment (smoors) [reply lib/vsc/administration/slurm/sync.py]: see https://github.com/hpcugent/vsc-administration/pull/104/commits/7f0c787575efddc3ba2f014b4e563971fd6661e7

**Pull request merged**